cli/scorecard/inventory.go

```diff
@@ -130,11 +146,12 @@ func (inventory *InventoryConfig) Export() error {
 		return err
 	}
 	err = inventory.exportToGcs(assetpb.ContentType_IAM_POLICY)
-	s.Stop()
-
 	if err != nil {
+		s.Stop()
```

Comment:
I'm not sure why this needs to be duplicated?

Reply:
removed in c317f93

Reply:
It's to remove the spinning message from console when each individual task runs into error